src/modExcelToXML.bas

```diff
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
```

Comment:
No this is not a good idea, I have implemented your nice error handler into this with last commit 916a66f30fe7319ec0c08ff1f8c8de869f1d88e6